x-pack/legacy/plugins/monitoring/public/components/chart/horizontal_legend.js

```diff
@@ -102,9 +97,9 @@ export class HorizontalLegend extends React.Component {
 
     return (
       <div className="monRhythmChart__legendHorizontal">
-        <div className="row monRhythmChart__legend-series">
-          { rows }
-        </div>
+        <EuiFlexGroup wrap={true} gutterSize="s" className="monRhythmChart__legend-series">
```

Comment:
Can you change the name of this class to match our naming schema. Likely should be `monRhythmChart__legendSeries`?